_articles/scripting/Typescript/typescript-introduction_rus.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Typescript введение
```

Comment:
Please keep the title in English, as it will appear in the sidebar, something like `Typescript Introduction (RUS)`?

Reply:
ok